sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Azure.Monitor.OpenTelemetry.LiveMetrics.csproj

```diff
@@ -1,15 +1,31 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Description>An OpenTelemetry .NET AzureMonitor OpenTelemetry Live Metrics</Description>
     <AssemblyTitle>AzureMonitor OpenTelemetry Live Metrics</AssemblyTitle>
     <Version>1.0.0-beta.1</Version>
     <PackageTags>Azure Monitor OpenTelemetry live Metrics ApplicationInsights</PackageTags>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <IncludeOperationsSharedSource>true</IncludeOperationsSharedSource>
+    <Nullable>enable</Nullable>
+    <DefineConstants>$(DefineConstants);LIVE_METRICS_EXPORTER;</DefineConstants>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
+    <PackageReference Include="OpenTelemetry" />
   </ItemGroup>
 
+  <!-- Shared sorce from Azure.Monitor.OpenTelemetry.Exporter -->
+  <ItemGroup>
+    <Compile Include="..\..\Azure.Monitor.OpenTelemetry.Exporter\src\Internals\ConnectionString\*.cs" LinkBase="Internals\ConnectionString" />
```

Comment:
Is there an msbuild variable to avoid the relative path?

Reply:
not a variable that i know about. This is what we're doing in all of our projects. I'll doublecheck the repo and see if there's anything i haven't considered

Reply:
I found something that will work. Let me follow up in a new PR